checkov/common/checks_infra/registry.py

```diff
@@ -21,6 +22,34 @@
 CHECKS_POSSIBLE_ENDING = {".json", ".yaml", ".yml"}
 
 
+class IACFrameworkTypes(str, Enum):
+    TERRAFORM = "terraform"
+    CLOUDFORMATION = "cloudformation"
+    SERVERLESS = "serverless"
+    KUBERNETES = "kubernetes"
+    SECRETS = "secrets"
+    TERRAFORM_PLAN = "terraform_plan"
+    ARM = "arm"
+    DOCKERFILE = "dockerfile"
+    DOCKER_IMAGE = "docker_image"
+    GITHUB_CONFIGURATION = "github_configuration"
+    GITLAB_CONFIGURATION = "gitlab_configuration"
+    KUSTOMIZE = "kustomize"
+    BICEP = "bicep"
+    OPENAPI = "openapi"
+    GITHUB_ACTION = "github_actions"
+    HELM = "helm"
+    GITLAB_CI = "gitlab_ci"
+    AZURE_PIPELINES = "azure_pipelines"
+    CIRCLECI_PIPELINES = "circleci_pipelines"
+    ANSIBLE = "ansible"
+
+
+IACFrameworksWithRunner = [IACFrameworkTypes.TERRAFORM, IACFrameworkTypes.CLOUDFORMATION, IACFrameworkTypes.KUBERNETES,
```

Comment:
why do you need this list? isn't it just the list of all enum values? you probably can gather it from `enum.Enum` internal methods

Reply:
This list contains only some of the enum values